src/events/qaMention.js

```diff
@@ -1,9 +1,13 @@
 const { Events } = require('discord.js');
 const { DISCORD_CONFIG } = require('../config');
+const { pollResults } = require('../utils/poll-vote');
 
 module.exports = {
   name: Events.MessageCreate,
   async execute(message) {
+
+    if (message.embeds[0]?.data?.type === 'poll_result') await pollResults(message)  
+
```

Comment:
Move this code to another event. Create a new file in events folder called pollVotationResults.js or something similar, then export a new event like this but only for poll results.